webextension/js/popup.js

```diff
@@ -2,6 +2,13 @@
 const CONTAINER_HIDE_SRC = "/img/container-hide.svg";
 const CONTAINER_UNHIDE_SRC = "/img/container-unhide.svg";
 
+function showPanel(panelSelector) {
+  for (let panelElement of document.querySelectorAll(".panel")) {
```

Comment:
We technically can, but since the statements in the loop modify a property of the object, I agreed with @bakulf that we should use `let` here to indicate that the variable does indeed change.